File: test_kle4/tloop52.pl
#!/usr/bin/perl

use strict;

sub fact {
 my $num=shift;
 if($num<1) {
	return 1;
 } else {
	my $f=$num*fact($num-1);
	return $f;
 }
}

 print fact(6),"\n";

 my $p=0.0;
 my $l=0;

 my $i;
 my $j;
 
 for($i=1;$i<11;$i++) {
	for($j=1;$j<800001;$j++) {
		$l = $i*$j+32;
		$p = $p+$l+fact(2);
	}
	print "loop $i: p = $p\n";
 };
 print ("result is : $p\n");

File: test_kle4/tloop52.py
#!/usr/bin/python

def fact(num):
	if num < 1:
		return 1
	else:
		f=num*fact(num-1)
		return f

print (fact(6))

p=0
r1 = range(1,800001,1)

for i in range(1,11):
	for j in r1:
		l=i*j+32
		p=p+l+fact(2)
	print ("loop ",i,"p=",p)
print ("result is : "+str(p))